### PR TITLE
feat: implement helm support

### PR DESCRIPTION
**Pull request by ComradeProgrammer**
feat: implement helm support

### CODE REVIEW

**Comment by hsluoyz** [comment]
@sagilio

**Comment by hsluoyz** [comment]
@sagilio

File: k8sgatekeeper/values.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default values for ksauth.
```
Comment (sagilio) [inline k8sgatekeeper/values.yaml:1]: Plz move some user options to here

**Review by sagilio — CHANGES_REQUESTED** [review]

**Comment by ComradeProgrammer** [comment]
@sagilio  revisions requested by last online mentor-student meeting have all been implemented in multiple PRs

**Comment by hsluoyz** [comment]
@sagilio plz review

**Review by sagilio — APPROVED** [review]
LGTM

**Pull request merged**